lib/sponsor/ui/sponsor_section.dart

```diff
@@ -115,23 +109,20 @@ class _SponsorsBuilder extends StatelessWidget {
           sponsorsList: sponsorsList,
           sponsorTypeTitle: sponsorTypeTitle,
         ),
+        mediumWidget: const SizedBox(),
```

Comment:
Sayfa küçüldüğü zaman widget bozulabilir, [responsive_builder.dart](https://github.com/flutterturkey/festival_flutterturkiye_org/pull/108/files#diff-851cf00e49062ff98d4b9a138d236c83ac26a977ed0d008e8b6db6a96f79b4bd) sayfasındaki düzenlemeleri yaparsak geri alabiliriz diye düşünüyorum.

Reply:
Düzenlemeleri geri almasam da şu anda aslında aynı işlevde, SizedBox() hiçbir şey göstermeyecek, bir yeri de kaplamıyor. Yani bozulma olmaz, yine de eskisine çevirmemi ister misin?

Reply:
If we make it SizedBox, it will be empty, isn't it?

Reply:
😄